services/core-web/common/utils/Validate.js

```diff
@@ -170,6 +176,9 @@ export const validateSelectOptions = memoize((data) => (value) => {
   }
 });
 
+export const integer = (value) =>
+  value && !Validate.checkInteger(value) ? "Invalid number." : undefined;
+
```

Comment:
Can you update this alert to be more descriptive? Ie "Invalid. The number must be a whole number, decimals not allowed." or something

Reply:
"Input must not contain decimals"?

Reply:
Fixed. Thanks :)